tests/src/OneLogin/Saml2/AuthTest.php

```diff
@@ -939,7 +938,7 @@ public function testLoginPOSTBindings()
         $settingsInfo['idp']['singleSignOnService']['binding'] = Constants::BINDING_HTTP_POST;
         $auth = new Auth($settingsInfo);
         $result = $auth->login(null, array(), false, false, true);
-        
+
```

Comment:
extra line

Reply:
I'm not sure. Looks like PHPStorm just took the spaces out of the line that's still there